src/Management/src/Prometheus/PrometheusExtensions.cs

```diff
@@ -39,19 +72,80 @@ public static IServiceCollection AddPrometheusActuator(this IServiceCollection s
         return services;
     }
 
+    /// <summary>
+    /// Adds the Steeltoe-configured OpenTelemetry Prometheus exporter to an <see cref="IApplicationBuilder" /> instance.
+    /// </summary>
+    /// <param name="builder">
+    /// The <see cref="IApplicationBuilder" />.
+    /// </param>
+    /// <returns>
+    /// The incoming <paramref name="builder" /> so that additional calls can be chained.
+    /// </returns>
     public static IApplicationBuilder UsePrometheusActuator(this IApplicationBuilder builder)
+    {
```

Comment:
Please add a test for this: `services.AddPrometheusActuator(configureMiddleware: false)` and set up the pipeline manually.